nexus/library/structure.py

```diff
@@ -766,6 +766,26 @@ def __init__(self,axes=None,scale=1.,elem=None,pos=None,mag=None,
     #end def __init__
 
 
+    def check_consistent(self,tol=1e-8,exit=True,message=False):
```

Comment:
The 'message' flag doesn't seem necessary, and could be removed to simplify the function.   The function can always return (consistent,msg).  If the caller doesn't want the message, it can ignore it (a single underscore is a valid variable name, so it's often used to represent an unwanted return value, giving something like "consistent, _ = self.systemcheck_consistent(exit=False)"

Similarly for check_folded_system.